<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenFileDialog</class>
 <widget class="QDialog" name="OpenFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open File</string>
  </property>
  <widget class="QPushButton" name="newFilePushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open an exisiting file ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uriPushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open file with URI</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameFile">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>20</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Plese give a name to the file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uri">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Plese input the URI to open the file</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>261</y>
     <width>791</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
